components/eam/src/physics/crm/crm_output_module.F90

```diff
@@ -88,6 +88,11 @@ module crm_output_module
       real(crm_rknd), allocatable :: qcltend (:,:)          ! CRM output tendency of cloud liquid water
       real(crm_rknd), allocatable :: qiltend (:,:)          ! CRM output tendency of cloud ice
 
+      real(crm_rknd), allocatable :: t_vt_tend (:,:)       ! CRM output tendency for variance transport
+      real(crm_rknd), allocatable :: q_vt_tend (:,:)       ! CRM output tendency for variance transport
+      real(crm_rknd), allocatable :: t_vt_ls   (:,:)       ! variance transport tendency
+      real(crm_rknd), allocatable :: q_vt_ls   (:,:)       ! variance transport tendency
+
```

Comment:
Comments could be more descriptive here.  Perhaps stating if they are tendencies from the crm -> gcm or gcm -> CRM. Also the comments should say what species these are the variance tendencies of.

Reply:
done.